Bottom: 069c168a45f7a42324a3b3f14c28870721b00b03
Top:    4e9ab32479c277bdf060713fca7c15ec6e714d2c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-29 15:53:44 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
